packages/http/src/rules/operation-reference-container-route.ts

```diff
@@ -0,0 +1,70 @@
+import { Operation, createRule, paramMessage } from "@typespec/compiler";
+import { getRoutePath } from "../route.js";
+import { OperationContainer } from "../types.js";
+
+export const operationReferenceContainerRouteRule = createRule({
+  name: "operation-reference-container-route",
```

Comment:
```suggestion
  name: "op-reference-container-route",
```

nit: would be a little nicer to use the short name for those?

Reply:
Yeah, it's a bit long